renderdoc/core/image_viewer.cpp

```diff
@@ -772,7 +776,7 @@ void ImageViewer::RefreshFile()
 
     float *rgba = (float *)data;
 
-    if (exrImage.images != NULL)
+    if(exrImage.images != NULL)
```

Comment:
Minor nit (and I only noticed because I was looking commit-wise to avoid the tinyexr commit, which normally I don't do) - could you squash formatting changes into the relevant commits so that each commit is formatted properly? I ask that each commit on its own is formatted (to avoid history noise) and compiles successfully (to help with bisects).